toolkit/resources/imageconfigs/additionalfiles/iso_initrd/installer/terminal-EULA.txt

```diff
@@ -0,0 +1 @@
+Sample EULA
```

Comment:
Suggestion: could you update this to something like "Sample terminal EULA" (and the other to "Sample GUI EULA") to make it very clear which EULA file is being shown?

Reply:
Done, thanks for the feedback!